components/raftstore/src/store/fsm/store.rs

```diff
@@ -1334,7 +1334,7 @@ pub fn create_raft_batch_system<EK: KvEngine, ER: KvEngine>(
 #[derive(Debug, PartialEq)]
 enum CheckMsgStatus {
     // The peer already exist
-    PeerExist,
+    PeerExist { force_send: bool },
```

Comment:
Seem `force_send` will always be `true`.

Reply:
Yes, it's always true in the current implementation. I think it would be clear if we explicitly state the message need to be force sent.